### ISSUE TITLE
PyData.sh Script Permission Error in Mac

### ISSUE THREAD

**Issue by vishnu-iyer** [donnemartin/dev-setup#73]
While running pydata.sh script the installation fails for virtualenv and virtualenvwrapper.
It says Permission Denied.

 Is anyone facing this issue.?

**Comment by donnemartin** [comment]
Hi @vishnu-iyer, can you post the output?

**Comment by iyervishnu** [comment]
Hi @donnemartin 
this is the output in the terminal after the installation is done.

 ```
/usr/bin/python:

 No module named virtualenvwrapper
virtualenvwrapper.sh: There was a problem running the initialization hooks.

If Python could not import the module virtualenvwrapper.hook_loader,
check that virtualenvwrapper has been installed for
VIRTUALENVWRAPPER_PYTHON=/usr/bin/python and that PATH is
set properly.

vishnuyr at Vishnus-MacBook-Air in ~
$
```

Also I have attached the Output of Running ./.dots .pydata and ./.dots bootstrap osxprep brew osx

[dots.log](https://github.com/donnemartin/dev-setup/files/1337256/dots.log)
[pydata.log](https://github.com/donnemartin/dev-setup/files/1337257/pydata.log)

**Comment by donnemartin** [comment]
Hmm, not sure why you're running into this issue.  Do these posts help?

* [Stack Overflow Post 1](https://stackoverflow.com/questions/29486113/problems-with-python-and-virtualenvwrapper-after-updating-no-module-named-virtu)
* [Stack Overflow Post 2](https://stackoverflow.com/questions/29149853/no-module-named-virtualenvwrapper)

**Comment by vishnu-iyer** [comment]
No none of this steps were helpful.

Regards,
Vishnu Iyer
+91-9833207793
Please consider your environmental responsibility. Before printing this e-mail message, ask yourself whether you really need a hard copy.
IMPORTANT: The contents of this email and any attachments are confidential. They are intended for the named recipient(s) only. If you have received this email by mistake, please notify the sender immediately and do not disclose the contents to anyone or make copies thereof.

**Comment by jpwalsh1** [comment]
I just got errors on this script too setting up a new 2017 macbook pro running 10.12.6.

I needed to update the script in this spot to the following. Also forced the py2 virtualenv to use the homebrew python too just to be safe. This section updating helped it not try to update system files that wanted me to run the file as root/etc.

```
export PIP_REQUIRE_VIRTUALENV=false
pip install pbr
pip install virtualenv
pip install --no-deps stevedore
pip install --no-deps virtualenvwrapper
```
It's odd because I've ran this file on another, older macbook, the same OS level and it ran fine.
